src/cat/cat.rs

```diff
@@ -456,7 +518,8 @@ fn write_nonprint_to_end<W: Write>(in_buf: &[u8], writer: &mut W, tab: &[u8]) ->
             128...159 => writer.write_all(&[b'M', b'-', b'^', byte - 64]),
```

Comment:
Can you change this to use byte literals (and maybe fix the other issues mentioned by the person on IRC)?